### ISSUE TITLE
An example of pure evaluation model's config file?

### ISSUE THREAD

**Issue by glouglouglousq** [tensorflow/models#6849]
Hello,

I've never seen an example of config file for doing only evaluation. I want to do a pure evaluation job, for example using 

> Running an Evaluation Job on CMLE

 of https://github.com/tensorflow/models/blob/master/research/object_detection/g3doc/running_on_cloud.md . 

Currently, I have only template of model config file for training, (faster_rcnn_resnet101_coco.config in config samples of object detection api) and the field

> 
> eval_input_reader: {
>   tf_record_input_reader {
>     input_path: ["path/to/record/file1.record",
                           "path/to/record/file2.record",
                           "..."]

in config.file is the **validation** set from what I understood and I can see some mAP and other metrics on it with tensorboard (I used Running a Multiworker (GPU) Training Job on CMLE for training in the same link above), but I want to perform an evaluation on a test .record file, what is the way to do in ML right (separate train/eval and test set) with my exported frozen graph.

Thanks you in advance. :)

OS platform: Ubuntu 18.04.2 LTS
tensorflow version : 1.13.1 installed from pip
Without CUDA/GPU 
@tensorflowbutler there is no command to reproduce here, it's just a question of finding a pure evaluation config file and a pure evaluation command to launch among the commands here: https://github.com/tensorflow/models/blob/master/research/object_detection/g3doc/running_on_cloud.md , if such a file exists and command exists.

**Comment by tensorflowbutler** [comment]
Thank you for your post. We noticed you have not filled out the following field in the issue template. Could you update them if they are relevant in your case, or leave them as N/A? Thanks.
What is the top-level directory of the model you are using
Have I written custom code
OS Platform and Distribution
TensorFlow installed from
TensorFlow version
Bazel version
CUDA/cuDNN version
GPU model and memory
Exact command to reproduce

**Comment by glouglouglousq** [comment]
Up please.

Still don't get how to do an evaluation job (not training with validation set, evaluation) on the cloud.

**Comment by tensorflowbutler** [comment]
Hi There,
 We are checking to see if you still need help on this, as this seems to be an old issue. Please update this issue with the latest information, code snippet to reproduce your issue and error you are seeing.
 If we don't hear from you in the next 7 days, this issue will be closed automatically. If you don't need help on this issue any more, please consider closing this.
